upgrade/1.5.2/README.md

```diff
@@ -354,11 +354,11 @@ SUCCESS
 These clusters are automatically backed up every 24 hours, but taking a manual backup at this stage in the upgrade
 enables restoring from backup later in this process if needed.
 
-### NCN reboot
+### NCN upgrade
 
-This is an optional step but is strongly recommended. As each patch release includes updated container images that may
-contain CVE fixes, it is recommended to reboot each NCN to refresh cached container images. For detailed instructions on
-how to gracefully reboot each NCN, refer to [Reboot NCNs](../../operations/node_management/Reboot_NCNs.md).
+This step is necessary so that nodes are using the correct images after running [Update NCN images](../1.5.2/README.md#update-ncn-images).
```

Comment:
The link to the upgrade page has not been added here. Is this fine?

Reply:
Great catch. I have added the link.